adapters/tappx/tappx.go

```diff
@@ -143,7 +148,7 @@ func (a *TappxAdapter) buildEndpointURL(params *openrtb_ext.ExtImpTappx, test in
 		thisURI = thisURI + "&ts=" + strconv.Itoa(int(t))
 	}
 
-	thisURI = thisURI + "&v=" + TAPPX_BIDDER_VERSION
+	thisURI = thisURI + "&v=" + TAPPX_BIDDER_VERSION + "&type_cnn=prebid"
 
```

Comment:
I know there are test cases that assert the `thisURI` expected values but, do you think it'd be more reliable or maintainable to use golangs URL package to manage queries and other URL related operations? This is entirely up to you.